meta-lxatac-software/recipes-devtools/qdl/qdl_git.bb

```
@@ -0,0 +1,25 @@
+SUMMARY = "Qualcomm DownLoader flashing tool"
+DESCRIPTION = "Communicate with Qualcomm SoCs to upload new software or \
+dump memory"
+HOMEPAGE = "https://github.com/linux-msm/qdl.git"
+SECTION = "devel"
+
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=da6bfde9cb5bc5120a51775381f6edf1"
+
+DEPENDS = "libxml2 libusb1"
+
+inherit pkgconfig
+
+SRCREV = "5db7794e9fdb73ed0c45384026cd8a62b5fff786"
+SRC_URI = "git://github.com/linux-msm/${BPN}.git;branch=master;protocol=https"
+
+PV = "0.0+${SRCREV}"
```

Shouldn't this be:

```suggestion
PV = "2.1+${SRCREV}"
```

Because [2.1](https://github.com/linux-msm/qdl/releases/tag/v2.1) is the most recent release and we have commits on top of it?
I honestly don't know though. This could be how git versions are usually handled.

---

Fixed and submitted to meta-qcom as well: https://github.com/qualcomm-linux/meta-qcom/pull/930